autoload/neomake/makers/ft/clojure.vim

```diff
@@ -0,0 +1,23 @@
+" vim: ts=4 sw=4 et
+
+function! neomake#makers#ft#clojure#EnabledMakers() abort
+    return ['clj_kondo']
+endfunction
+
+function! neomake#makers#ft#clojure#clj_kondo() abort
+    let maker = {
+        \ 'exe': 'clj-kondo',
+        \ 'args': ['--lint'],
+        \ 'errorformat':
+            \ '%f:%l:%c: Parse %t%*[^:]: %m,'.
+            \ '%f:%l:%c: %t%*[^:]: %m,'.
+            \ '%-Glinting took %.%#'
```

Comment:
This was just reformatted, and I've added the last item to ignore the timing information - or do you think it is useful?

Reply:
It's not that useful, no.  I just didn't know an easy way to strip it, I was worried I'd have to run it through a vimscript function or something to detect it.  Your solution looks good.